common/web/gesture-recognizer/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@keymanapp/gesture-recognizer",
+  "description": "The core gesture-recognition engine used by Keyman's Web-based OSKs.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/keymanapp/keyman.git"
+  },
+  "keywords": [
+    "input",
+    "languages",
+    "keyboards"
+  ],
+  "author": "SIL International",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/keymanapp/keyman/issues"
+  },
+  "homepage": "https://github.com/keymanapp/keyman#readme",
```

Comment:
```suggestion
```

Omit these details. This is a private dependency and this information just adds unnecessary noise to package.json. Smaller is better here.